Applications/Spire/Source/UiViewer/SecurityWidgetTestWidget.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Spire/UiViewer/SecurityWidgetTestWidget.hpp"
+#include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Spire/Utility.hpp"
+#include "Spire/Ui/FlatButton.hpp"
+
+using namespace Beam;
+using namespace Nexus;
+using namespace Spire;
+
+SecurityWidgetTestWidget::SecurityWidgetTestWidget(QWidget* parent)
+    : QWidget(parent),
+      m_security_widget(nullptr) {
+  auto container = new QWidget(this);
+  m_layout = new QGridLayout(container);
+  auto info_label = new QLabel(tr("Click SecurityWidget to set focus."));
+  info_label->setAlignment(Qt::AlignCenter);
+  m_layout->addWidget(info_label, 0, 0, 1, 2);
+  m_dark_theme_check_box = make_check_box(tr("Dark Theme"), this);
+  m_layout->addWidget(m_dark_theme_check_box, 1, 0);
+  auto reset_button = make_flat_button(tr("Reset"), this);
+  reset_button->setFixedSize(scale(100, 26));
+  reset_button->connect_clicked_signal([=] { on_reset_button(); });
+  m_layout->addWidget(reset_button, 1, 1);
+  m_status_label = new QLabel(this);
+  m_layout->addWidget(m_status_label, 2, 0, 1, 2);
+  m_model.add(SecurityInfo(
```

Comment:
Write a function to return test model that is used across all of these widgets. Include that function in ```Spire/SecurityInput/LocalSecurityInputModel.hpp```

Reply:
Do you want this applied to every widget, or just the test widgets?

Reply:
Do what you think is best.